lib/start.js

```diff
@@ -32,6 +32,10 @@ function start(_opts, _cb) {
     opts.spawnCb = noop;
   }
 
+  if (defaultConfig.version.startsWith('4.')) {
```

Comment:
Don't use default config for version detection

Reply:
using opts now